### PR TITLE
Dark mode updated

### PR DESCRIPTION
**Pull request by daundkar-ank**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,27 @@
...
+      document.body.style.backgroundColor = "black";
+    } else {
+      setMode("light");
+      document.body.style.backgroundColor = "white";
+    }
+  };
+
+  return (
+    <>
+      <Navbar title="Home" subtitle="Dashboard" mode={mode} toggleMode={toggleMode} />
```
Comment (kumarparth380) [inline src/App.js]: check if your prettier format is working or not

File: src/components/typography/Card.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { PropTypes } from "prop-types";
+
+const Card = (props) => {
+  console.log(props.Author);
```
Comment (kumarparth380) [inline src/components/typography/Card.js]: try to remove console whenever you raise a PR

File: src/components/typography/CardHolder.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import Card from "./Card";
+import { PropTypes } from "prop-types";
+
+const CardHolder = (props) => {
+  const src = [
```
Comment (kumarparth380) [inline src/components/typography/CardHolder.js]: don't use static data. Try to use the API call for the data.

**Review by kumarparth380 — CHANGES_REQUESTED** [review]
I wanted you to make a context for the Theme rather than drilling it as a props in the component tree.

File: src/App.js (hunk 1 above)
Comment (daundkar-ank) [reply src/App.js]: Prettier format is working fine

File: src/App.js [hunk 4]
```diff
@@ -19,15 +20,17 @@ function App() {
 
   return (
     <>
-      <themes.Provider value={toggleMode}>
-        <Navbar title="Home" subtitle="Dashboard" mode={mode} />
-        <CardHolder />
-      </themes.Provider>
+      <Themes.Provider value={toggleMode}>
```
Comment (kumarparth380) [inline src/App.js:23]: <ThemesProvider>
       <DefaultTheme.Provider value={mode}>
          <Navbar title="Home" subtitle="Dashboard" />
          <CardHolder />
        </DefaultTheme.Provider>
        </ThemesProvider>
        
        
        themesProvider.js
        
        export const ThemesProvider =  ({children}) => {
        return (
        <Themes.Provider value={toggleMode}>
        {children}
        </Themes.Provider>
        )
        }

**Review by kumarparth380 — APPROVED** [review]

**Review by kumarparth380 — APPROVED** [review]

**Pull request merged**